.travis.yml

```diff
@@ -1,7 +1,10 @@
+sudo: required
```

Comment:
@llorllale Is sudo really required?

Reply:
@amihaiemil I'm not entirely sure (haven't tested), but the [Travis Docs](https://docs.travis-ci.com/user/docker/) seem to include it always

Reply:
@llorllale I see, let's leave it like that then